src/modules/api.js

```diff
@@ -0,0 +1,40 @@
+const successMessage = document.getElementById('success-message-id');
+class API {
+  constructor() {
+    this.gameId = 'hOOJqAq1A3QpeN0xQUs2';
+    this.url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+  }
+
+     addLeader = async (user, score) => {
+       const response = { score, user }; await fetch(`${this.url + this.gameId}/scores`, {
+         method: 'POST',
+         body: JSON.stringify(response),
+         headers: {
+           Accept: 'application/json',
+           'Content-Type': 'application/json',
+         },
+       })
+         .then(async (response) => {
+           const scoreResponse = await response.json();
+           successMessage.textContent = scoreResponse.result;
```

Comment:
- [ ] [Optional] 👍🎉 Great job on implementing the success message! 🌟 However, it seems that the message is overflowing on some screens, which could affect the user experience.🔍📏 To fix this issue, I recommend adjusting the styling of the success message to ensure it fits properly on all screen sizes. You can try reducing the font size, adjusting the padding, or adding some line breaks if needed.

![image](https://github.com/yusufyusie/Leaderboard/assets/55840999/30b77dd9-1acf-4028-8fba-5bdfd6862f5b)

💻🚀 By making these adjustments, you'll ensure that the success message is displayed correctly and doesn't disrupt the user interface. Keep up the good work, and I'm confident your attention to detail will make your project shine! 💪